targets/rpi3.ld

```diff
@@ -0,0 +1,46 @@
+/*
+ * This file assumes that you are running with the Raspberry PI 3/4 firmware
+ * that expects bare metal to be linked for a boot address of 0x80000.
+ *
+ * You may want to consult https://www.raspberrypi.org/documentation/configuration/config-txt/boot.md
+ * if you want to see how to force this boot address in your config.txt.
+ */
+
+SECTIONS
+{
+    . = 0x80000;
+    .text : { KEEP(*(.text.boot)) *(.text .text.* .gnu.linkonce.t*) }
+    .rodata : { *(.rodata .rodata.* .gnu.linkonce.r*) }
+    PROVIDE(_data = .);
+
+		/* Globals with initial value */
```

Comment:
This file uses inconsistent whitespace. I don't care much which variant you pick (tabs or spaces), but please keep it consistent.

Reply:
Yeah, my editor (goland) didn't know about that type of file so things were not consistent.  I used the goland "convert spaces to tabs" to fix it.